docs/src/user_docs/assembly/io_operations.md

```diff
@@ -12,11 +12,11 @@ Miden assembly provides a set of instructions for moving data between the operan
 | ------------------------------------------------------------------------- | ----------- | ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | push.*a* <br> - *(1-2 cycles)* <br> push.*a*.*b* <br> push.*a*.*b*.*c*... | [ ... ]     | [a, ... ] <br> [b, a, ... ] <br> [c, b, a, ... ] | Pushes values $a$, $b$, $c$ etc. onto the stack. Up to $16$ values can be specified. All values must be valid field elements in decimal (e.g., $123$) or hexadecimal (e.g., $0x7b$) representation. |
 
-When specifying values in hexadecimal format, it is possible to omit the periods between individual values as long as total number of specified bytes is a multiple of $8$. That is, the following are semantically equivalent:
+The value can be specified in hexadecimal format without periods between individual values as long as it specifying a Word, i.e. total number of specified bytes in the value is equal to $32$. Note that hexadecimal values devided by periods (short hexadecimal strings) assumed to be encoded in big-endian format, while the strings specifying the whole Word without periods (long hexadecimal strings) assumed to be encoded in little-endian format. That is, the following are semantically equivalent:
```

Comment:
minor grammar and wording changes:

The value can be specified in hexadecimal form without periods between individual values as long as it describes a full word ($4$ field elements or $32$ bytes). Note that hexadecimal values separated by periods (short hexadecimal strings) are assumed to be in big-endian order, while the strings specifying whole words (long hexadecimal strings) are assumed to be in little-endian order. That is, the following are semantically equivalent: